Classes/RedirectStorage.php

```diff
@@ -138,17 +149,19 @@ protected function addRedirectByHost($sourceUriPath, $targetUriPath, $statusCode
     {
         $redirect = new Redirect($sourceUriPath, $targetUriPath, $statusCode, $host);
         $this->updateDependingRedirects($redirect);
+        $this->persistenceManager->persistAll();
```

Comment:
ah i was too slow :)

Just wanted to mention, that I've read several times that `persistAll()` could be evil.

We can do this instead:
https://github.com/neos/redirecthandler-databasestorage/blob/master/Classes/Domain/Repository/RedirectRepository.php#L225